frontend/react/style-guide.md

```diff
@@ -177,22 +177,18 @@ src
     // good
     const reservationItem = <ReservationCard />;
     ```
-  - **Component Hierarchy**: 
+  - **Component Hierarchy**:
     - Component files should be inside folders that match the component's name.
     - Use index.js as the filename of a container component. Use `Container` as the suffix of the component's name.
     - Use layout.js as the filename of a layout component.
```

Comment:
in this section, this patter we already dont use it. maybe can be good idea talk about  sub components own of the Main component here.